packages/mgt-components/src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -900,9 +900,9 @@ export class MgtPeoplePicker extends MgtTemplatedTaskComponent {
        <mgt-person
          class="person-image-selected"
          .personDetails=${person}
-         .fetchImage=${!this.disableImages}
-         .view=${ViewType.oneline}
-         .personCardInteraction=${this.personCardInteraction}
+         ?fetch-image=${!this.disableImages}
+         view="oneline"
+         person-card="none"
```

Comment:
Pretty sure we want this one correctly data bound to this.personCardInteraction

Reply:
Definitely! Missed that error.